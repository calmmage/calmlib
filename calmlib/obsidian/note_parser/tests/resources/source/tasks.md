- 01
    - some task
- 02
    - more ideas
- 03
    - even more text
