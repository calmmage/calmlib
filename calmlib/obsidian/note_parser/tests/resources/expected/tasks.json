{
  "items": [
    {
      "type": "task",
      "text": "01",
      "children": [
        {
          "type": "list_item",
          "text": "some task"
        }
      ]
    },
    {
      "type": "task",
      "text": "02",
      "children": [
        {
          "type": "list_item",
          "text": "more ideas"
        }
      ]
    },
    {
      "type": "task",
      "text": "03",
      "children": [
        {
          "type": "list_item",
          "text": "even more text"
        }
      ]
    }
  ]
}